api/dp-openapi.yaml

```diff
@@ -1290,6 +1290,49 @@ paths:
               schema:
                 $ref: '#/components/schemas/round_stats'
 
+  /documents:
+    get:
+      tags:
+      - documents
+      description: Gets an array of document names, if lang_code is given then only those available in that language
+      parameters:
+      - name: lang_code
+        in: query
+        description: language code
+        schema:
+          type: string
+      responses:
+        200:
+          description: Array of document names
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: string
+
+  /documents/{document_name}:
```

Comment:
I would make this `/documents/{document}:` vs `document_name` since we return a filename and not the name of the document itself.

Reply:
good point.